meta-lmp-base/recipes-core/initrdscripts/initramfs-module-install-efi_%.bbappend

```diff
@@ -2,4 +2,4 @@ FILESEXTRAPATHS:prepend := "${THISDIR}/${PN}:"
 
 # Prefer gptfdisk instead of parted
 RDEPENDS:${PN}:remove = "parted dosfstools"
-RDEPENDS:${PN} += "efibootmgr gptfdisk"
+RDEPENDS:${PN} += "efibootmgr gptfdisk systemd-crypt"
```

Comment:
`RDEPENDS:${PN} += "efivar"` to use the efivar tool on the ramdisk

Reply:
absolutely! makes sense - I shouldnt have followed the code that we have in LmP :) . will refactor that as well